### PR TITLE
SSH instructions

### PR DESCRIPTION
**Pull request by guilhermedemouraa**

### CODE REVIEW

**Review by mihelich — COMMENTED** [review]
Thanks @guilhermedemouraa, generally looks really good. I made some comments, most importantly around SSH configuration - as written I think this works for us, but not for customers.

File: website/docs/ssh/ssh.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+id: ssh
+title: Access and Develop on the Brain
+---
+
+Users can remotely access and start developing on the brain via `SSH` access.
+The only requirement is to be connected to the same network of your Amiga.
+
+### Sign up for a farm-ng-user account
+
+#### 1. Request SSH access to our Fleet Customer Support via email (fleet@farm-ng.com)
```
[the remark this answers is not in the document]
Comment (guilhermedemouraa) [reply website/docs/ssh/ssh.md:11]: No, I meant to leave it as a placeholder. Let me change it and we can fix later once we have a permanent solution.

File: website/docs/ssh/ssh.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+#### 4. Click on the first option that pops up (/home/user/.ssh/config)
+
+![Screenshot from 2023-10-05 13-58-05](https://github.com/farm-ng/amiga-dev-kit/assets/39603677/6a577080-a052-4a90-9ffb-491d7bb7c2ed)
+
+#### 5. Add the following lines to your file
+
+```bash
+Host boron-banana
+  HostName boron-banana
```
[the remark this answers is not in the document]
Comment (mihelich) [reply website/docs/ssh/ssh.md]: OK - I added https://github.com/farm-ng/workspace/issues/1887 to our sprint. We should at least get that into a release next week.

**Review by mihelich — APPROVED** [review]
Approved, pending fix to use IP address.

File: .vscode/settings.json [hunk 3]
```diff
@@ -5,5 +5,6 @@
         "~/.local/lib/python3.8/site-packages"
     ],
     "python.analysis.autoImportCompletions": false,
-    "python.formatting.provider": "black"
+    "python.formatting.provider": "black",
+    "cmake.sourceDirectory": "/home/gdemoura/amiga-dev-kit/farm-ng-amiga"
```
Comment (Hackerman342) [inline .vscode/settings.json:9]: Don't push stuff like this that is specific to your machine/user!

Comment (guilhermedemouraa) [reply .vscode/settings.json:9]: Sorry I didn't see this getting added to the commit. I probably missed it bc I was rushing to publish this doc for Jeff's call last Friday. I will remove it in a future PR.

Comment (Hackerman342) [reply .vscode/settings.json:9]: No worries, doing it in #214 

If you're constantly having to ignore that change (which you shouldn't be) we can adjust settings either for your vs code or using .gitignore

**Pull request merged**